enterprise/coderd/coderd.go

```diff
@@ -339,6 +339,15 @@ func New(ctx context.Context, options *Options) (_ *API, err error) {
 				r.Get("/", api.groupByOrganization)
 			})
 		})
+		r.Route("/provisionerkeys", func(r chi.Router) {
+			r.Use(
+				httpmw.ExtractProvisionerDaemonAuthenticated(httpmw.ExtractProvisionerAuthConfig{
```

Comment:
From my understanding we do not want this endpoint to be usable with PSK - which is why I've not filled it here.